### PR TITLE
Tm 298 cprobe config

### PR DESCRIPTION
**Pull request by zwright**

### CODE REVIEW

File: instance-configs/Config.cprobe.groovy [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+ * NOTE
+ * ----
+ * This configuration assumes that the development environment will be used with
+ * run-app and the production environment will be used with the application
+ * packaged as a WAR and deployed to tomcat. Running grails run-war or otherwise
+ * running a WAR with the development profile set up or activating the
+ * production environment when running grails run-app are scenarios that have
+ * NOT been tested.
+ */
+quickStartURL = "../../files/NEPTUNE_tranSMART_Quick_Start_Guide.pdf"
```
Comment (rlreamy) [inline instance-configs/Config.cprobe.groovy]: Should we point this at a CPROBE quickstart guide?

File: instance-configs/Config.cprobe.groovy [hunk 2]
```diff
@@ -0,0 +1,716 @@
...
+
+/* {{{ Personalization */
+// application logo to be used in the login page
+com.recomdata.largeLogo = "cprobeLogo.png"
+
+// application logo to be used in the search page
+com.recomdata.searchtool.smallLogo="cprobeLogo.png"
+
+// contact email address
+com.recomdata.contactUs = "mailto:NeptuneTMSupport@umich.edu"
```
Comment (rlreamy) [inline instance-configs/Config.cprobe.groovy]: Guessing we should change both of the contactUs addresses here.

File: instance-configs/Config.cprobe.groovy [hunk 3]
```diff
@@ -0,0 +1,716 @@
...
+com.recomdata.searchtool.contactUs = "mailto:roesch@med.umich.edu"
+
+// administrator email address
+com.recomdata.administrator="roesch@med.umich.edu"
+
+// application title
+com.recomdata.appTitle = "C-Probe tranSMART" // v + org.transmart.originalConfigBinding.appVersion +  " (GPL, PostgresSQL)"
+
+//Quick Start Guide URL
+quickStartURL = "../../files/NEPTUNE_tranSMART_Quick_Start_Guide.pdf"
```
Comment (rlreamy) [inline instance-configs/Config.cprobe.groovy:183]: Another reference to the quickstart guide (not sure why there are two).

**Review by rlreamy — CHANGES_REQUESTED** [review]

File: instance-configs/Config.cprobe.groovy (hunk 1 above)
Comment (zwright) [reply instance-configs/Config.cprobe.groovy]: Don't think there is one. Let's check with Becky.

File: instance-configs/Config.cprobe.groovy (hunk 2 above)
Comment (zwright) [reply instance-configs/Config.cprobe.groovy]: This is the same as production. Let's check with Becky.

**Review by rlreamy — APPROVED** [review]

**Pull request merged**